cmake/DD4hepCPack.cmake

```diff
@@ -0,0 +1,46 @@
+
+set(CPACK_PACKAGE_NAME "dd4hep")
+set(CPACK_SET_DESTDIR TRUE)
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://github.com/AIDASoft/dd4hep")
+set(CPACK_PACKAGE_DESCRIPTION "Detector Description Toolkit for HEP")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY ${CPACK_PACKAGE_DESCRIPTION})
+set(CPACK_PACKAGE_VENDOR "AIDASoft")
+set(CPACK_PACKAGE_VERSION ${dd4hep_VERSION})
```

Comment:
```suggestion
set(CPACK_PACKAGE_VERSION ${DD4hep_VERSION})
```
?

Reply:
Ah, actually there is no variable `${DD4hep_VERSION}` either. but because of the following lines this one isn't needed, so I removed it.

Reply:
This
https://github.com/AIDASoft/DD4hep/blob/fbb34839feea6f0462a1c55fad9ba1c21584b6ee/CMakeLists.txt#L98-L102
with this
https://github.com/AIDASoft/DD4hep/blob/fbb34839feea6f0462a1c55fad9ba1c21584b6ee/cmake/DD4hepBuild.cmake#L208
doesn't create DD4hep_VERSION?